.size-limit.js

```diff
@@ -0,0 +1,132 @@
+module.exports = [
+  {
+    "webpack": false,
+    "path": ["packages/koa-liveness-ping/dist"],
```

Comment:
We don't need `size-limit` for the server side or test packages. We should remove all the test and koa libraries from this.